openslides/assignments/access_permissions.py

```diff
@@ -10,18 +10,6 @@ class AssignmentAccessPermissions(BaseAccessPermissions):
     """
     base_permission = 'assignments.can_see'
 
-    def get_serializer_class(self, user=None):
```

Comment:
Maybe you also want to import the serializer in the ready() in apps.py?

Reply:
done (it is not needed, because it is imported indirectly by importing .views, but you are sure that this is better for the future)